src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -641,7 +641,7 @@ async Task<EntityQueryResult> IDurableEntityClient.ListEntitiesAsync(EntityQuery
                 var result = await entityBackendQueries.QueryEntitiesAsync(
                     new EntityBackendQueries.EntityQuery()
                     {
-                        InstanceIdStartsWith = query.EntityName != null ? $"${query.EntityName}" : null,
+                        InstanceIdStartsWith = query.EntityName != null ? $"@{query.EntityName.ToLowerInvariant()}@" : null,
```

Comment:
Will this cause any issues with new durable functions?

Reply:
not in this case... `EntityName` is already the entity name only, it was defined a bit differently for that query.